cns/configuration/configuration.go

```diff
@@ -52,6 +52,7 @@ type CNSConfig struct {
 	WatchPods                   bool `json:"-"`
 	WireserverIP                string
 	GRPCSettings                GRPCSettings
+	EnableK8sDevicePlugin       bool
```

Comment:
please, I have sorted this so many times

Reply:
ok moved it up

Reply:
@jaer-tsun Bonus points: add a comment to the bottom of this struct saying something to the effect of "keep this list alphabetically sorted," so someone doesn't make the same mistake in the future.